assets/sass/utilities/_font-weight.scss

```
@@ -0,0 +1,25 @@
+/**
+ * Font Weight utilities.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-font-weight-bold {
+		font-weight: 1000 !important;
```

This is heavier than `bold` (which is `700`). We can simply use the keyword value instead.
```suggestion
		font-weight: bold !important;
```

---

We can't use bold (my original choice) because of the linter: `Expected numeric font-weight notation`. Updated to 700.

---

In this case I think a linter rule-ignore line would be fine, but no need to change.